smdebug/tensorflow/keras.py

```diff
@@ -263,27 +263,36 @@ def _is_output_layer(self, mode, layer_outputs):
                 model_outputs.extend(per_replica_model.outputs)
         else:
             model_outputs.extend(model.outputs)
+        # In TF 2.X, calling `layer_outputs[0] in model_outputs gives the error:
```

Comment:
purpose of this comment?

Reply:
this comment was added before it was determined that in tf 2.x, we cant get intermediate input and output values. we can remove the comment